packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/stackinstances-action.ts

```diff
@@ -0,0 +1,97 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { Action } from '../action';
+import { validatePercentage } from '../common';
+import { SingletonPolicy } from './private/singleton-policy';
+import { CommonCloudFormationStackSetOptions, StackInstances, StackSetParameters } from './stackset-types';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core';
+
+/**
+ * Properties for the CloudFormationDeployStackInstancesAction
+ */
+export interface CloudFormationStackInstancesActionProps extends codepipeline.CommonAwsActionProps, CommonCloudFormationStackSetOptions {
```

Comment:
I believe this should be called `CloudFormationDeployStackInstancesActionProps`.

Reply:
Oh whoops. Good catch.